### ISSUE TITLE
'LineStyle','none'  does not work with "fill" function

### ISSUE THREAD

**Issue by pfbuxton** [matlab2tikz/matlab2tikz#896]
Hello,

If you run this code the pdf will have lines along the filled area. Note, in version-0.6.0 the fill command works as expected without the lines.

```matlab
clear;
figure;

theta = linspace( 0.0, 2.0*pi, 20 );
fill( cos(theta), sin(theta), 'Red', 'LineStyle','none' );

matlab2tikz( 'test.tikz', 'standalone',true );
system( 'pdflatex test.tikz' );
```

Thanks,
Peter

**Comment by egeerardyn** [comment]
I can reproduce this in R2014a and R2014b. Thank you for reporting this!

**Comment by miscco** [comment]
It seems that we have to add a check for linewidth in [m2t, str] = drawPatch(m2t, handle)

I would suggest the following
```matlab
        if isLineVisible(handle)
        [m2t, drawOptions] = setColor(m2t, handle, drawOptions, 'draw', ...
                                         s.edgeColor, 'none');
       end
        [m2t, drawOptions] = setColor(m2t, handle, drawOptions, 'fill', ...
                                         s.faceColor);
```

**Comment by miscco** [comment]
Meh doesnt work, one has to set the edge color to none ...

**Comment by miscco** [comment]
I opened a PR that should fix that #898

**Comment by egeerardyn** [comment]
I can confirm (both R2014a and R2014b) that this is fixed by #898.
